CMakeLists.txt

```diff
@@ -3,12 +3,24 @@
 # CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
 
 cmake_minimum_required(VERSION 2.8.7)
+PROJECT(GazeboYARPPlugins)
 
-PROJECT(GazeboYarpPlugins)
+# Project version
+set(VARS_PREFIX ${PROJECT_NAME})
+set(${VARS_PREFIX}_MAJOR_VERSION 2)
+set(${VARS_PREFIX}_MINOR_VERSION 3)
+set(${VARS_PREFIX}_PATCH_VERSION 68)
+set(${PROJECT_NAME}_VERSION
+    ${${VARS_PREFIX}_MAJOR_VERSION}.${${VARS_PREFIX}_MINOR_VERSION}.${${VARS_PREFIX}_PATCH_VERSION})
 
-find_package(YARP 2.3.64.13 REQUIRED)
+message (STATUS "${PROJECT_NAME} version ${${PROJECTNAME}_VERSION}")
```

Comment:
It may be a personal preference, but I prefer to avoid adding output to CMake unless strictly necessary.